sql/src/planner.rs

```diff
@@ -106,6 +106,9 @@ pub enum Error {
     #[snafu(display("Tag column not found, name:{}", name))]
     TagColumnNotFound { name: String },
 
+    #[snafu(display("Timestamp column can not be Tag, name:{}", name))]
+    TimestampColumnCanNotTag { name: String },
```

Comment:
A better name may be `TimestampKeyTag` (Timestamp column is different from timestamp key):
```suggestion
    #[snafu(display("Timestamp key column can not be tag, name:{}.\nBactrace:\n{:?}", name, backtrace))]
    TimestampKeyTag { name: String, backtrace: Backtrace },
```

Reply:
OK